test/e2e/admin_portal.go

```diff
@@ -197,4 +197,45 @@ var _ = Describe("Admin Portal E2E Testing", func() {
 			SaveScreenshotAndExit(wd, err)
 		}
 	})
+
+	It("Should be able to navigate to other regions", func() {
+		NUMBER_OF_REGIONS := 40
```

Comment:
Will this assumption for region number be valid for environments and clouds, like INT and Fairfax? From my own experiences with the ImageConfig controller (see #2340) e2e tests will still run in those environments but variables such as `AZURE_ENVIRONMENT` will be different, which affects the list we expect to get back.

Reply:
I believe these tests still only run in development as there are variables like this which currently make it difficult to test properly in production, int and FF